rt_gene/cfg/ModelSize.cfg

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+PACKAGE = "rt_gene"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+gen.add("model_size", double_t, 0, "Scale of the 3D model of the face used for head pose estimation", 30.0, 1.0, 100.0)
```

Comment:
Is that a reasonable range? That probably account from mice to elephant interpupillary distances currently doesn't it? :P

Reply:
Agreed too...